cluster/vagrant-kubernetes/setup_master.sh

```diff
@@ -30,7 +30,14 @@ systemctl enable cockpit.socket && systemctl start cockpit.socket
 rm -rf /var/lib/kubelet
 
 # Create the master
-kubeadm init --pod-network-cidr=10.244.0.0/16 --token abcdef.1234567890123456
+kubeadm init --config <(cat <<EOF)
```

Comment:
@stu-gott are you sure this works? I am pretty sure you wanted to do

```bash
cat > /etc/kubernetes/kubeadm.conf <<EOF
apiVersion: kubeadm.k8s.io/v1alpha1
kind: MasterConfiguration
apiServerExtraArgs:
  runtime-config: admissionregistration.k8s.io/v1alpha1
token: abcdef.1234567890123456
pod-network-cidr: 10.244.0.0/16
kubernetes-version: v1.9.3
EOF

kubeadm init --config /etc/kubernetes/kubeadm.conf
```

Reply:
And here the config which finally works:

```bash
cat >/etc/kubernetes/kubeadm.conf <<EOF
apiVersion: kubeadm.k8s.io/v1alpha1
kind: MasterConfiguration
apiServerExtraArgs:
  runtime-config: admissionregistration.k8s.io/v1alpha1
token: abcdef.1234567890123456
networking:
  podSubnet: 10.244.0.0/16
EOF

kubeadm init --config /etc/kubernetes/kubeadm.conf
```

Reply:
@stu-gott note the difference between `pod-network-cidr` and `podSubnet`.

Reply:
Hrm. Good catch. I'm not sure how that parentheses got misplaced, but minus the syntax error the original approach does work. Why create/keep a cruft file on the filesystem? As far as I know attempting to run `kubeadm init` again after the initial attempt is doomed to fail, is it not?

Reply:
Found it. The cluster-sync process moved the parenthesis and created the syntax error. Why are scripts modifying scripts? That feels very wrong!

Reply:
Maybe shfmt (the shell formatter) has issues with that construct, but anyway we already have that part merged. Just  remove that part. Don't worry about the file.